src/app/core/auth.service.ts

```diff
@@ -314,6 +315,12 @@ export class AuthService {
     return this.group.getValue() > AUTH_GROUP.USER;
   }
 
+  /**
+   * Identifies if the user should be able to see the "revise" button on the action panel of a learning object.
```

Comment:
Instead of commenting this function from the perspective of where it is used at the moment, we should document it in a way that would still make sense if used for another reason. This also applies to the name of the function. Perhaps something like `hasEditorPrivilege` would abstract it away from the action panel a bit better?